loader/standard_loader.py

```diff
@@ -1,11 +1,62 @@
+              'json_schema/cgp/gen3/2.0.0/cgp_gen3_metadata.json')
+
+
 class StandardFormatBundleUploader:
     def __init__(self, dss_uploader: DssUploader, metadata_file_uploader: MetadataFileUploader) -> None:
         self.dss_uploader = dss_uploader
         self.metadata_file_uploader = metadata_file_uploader
 
-    def load_all_bundles(self, input_json: dict):
-        # TODO Implement this loader
-        raise NotImplemented
+    @staticmethod
+    def _parse_bundle(bundle: dict) -> tuple:
+        data_bundle = bundle['data_bundle']
+        bundle_uuid = data_bundle['id']
+        metadata_dict = data_bundle['user_metadata']
+        data_objects = bundle['data_objects']
+        return bundle_uuid, metadata_dict, data_objects
+
+    @staticmethod
+    def _get_cloud_urls(file_info: dict):
+        return {url_dict['url'] for url_dict in file_info['urls']}
+
+    @staticmethod
+    def _get_file_ids(file_guid: str):
+        return file_guid.split('/')[1]
+
+    def _load_bundle(self, bundle: dict):
+        bundle_uuid, metadata_dict, data_objects = self._parse_bundle(bundle)
+        file_info_list = []
+
+        # load metadata
+        file_uuid, file_version, filename = \
+            self.metadata_file_uploader.load_dict(metadata_dict,
+                                                  "metadata.json",
+                                                  SCHEMA_URL,
+                                                  bundle_uuid)
+        file_info_list.append(dict(uuid=file_uuid, version=file_version, name=filename, indexed=True))
+
+        # load data files by reference
+        for file_guid in data_objects:
+            file_info = data_objects[file_guid]
+            cloud_urls = self._get_cloud_urls(file_info)
+            file_uuid = self._get_file_ids(file_guid)
+            file_uuid, file_version, filename = \
+                self.dss_uploader.upload_cloud_file_by_reference(file_info['name'],
+                                                                 # use did for uuid for now. will probably have to
+                                                                 # extract from guid (did) in the future
+                                                                 file_uuid,
+                                                                 cloud_urls,
+                                                                 bundle_uuid,
+                                                                 file_guid,
+                                                                 file_version=file_info["updated"])
+            file_info_list.append(dict(uuid=file_uuid, version=file_version, name=filename, indexed=False))
+
+        # load bundle
+        self.dss_uploader.load_bundle(file_info_list, bundle_uuid)
+
+    def load_all_bundles(self, input_json: list):
+        for bundle in input_json:
+            self._load_bundle(bundle)
```

Comment:
It's a great idea to include all of this information! It will be vital when doing large uploads. 

Dumping all of that info to stdout by default seems a little unwieldy to me. How to you feel about having a command line option to dump that info to files instead?